clouddq/main.py

```diff
@@ -512,6 +520,12 @@ def main(  # noqa: C901
                 sql_string=sql_string,
                 dbt_model_path=dbt_rule_binding_views_path,
             )
+            dq_target_rule_binding_configs_dict[
+                f"{rule_binding_id}_failed_records_sql_string"
+            ] = failed_records_sql_string
```

Comment:
please check with sandeep on the rule-binding <> rule logic. 1) should a failed record query for a rule return failed record for every other rules in the same rule_binding? 2) if a rule_binding has 100 rules should the same failed record query be persisted 100 times across all of the rules in the same rule_binding?